src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3942,8 +3932,10 @@ private bool HasTopLevelNullabilityConversion(TypeWithAnnotations source, TypeWi
             ParameterSymbol target = null,
             bool reportTopLevelWarnings = true,
             bool reportRemainingWarnings = true,
-            bool extensionMethodThisArgument = false)
+            bool extensionMethodThisArgument = false,
+            bool trackNullability = false)
```

Comment:
>trackNullability [](start = 17, length = 16)

nit: consider `trackState` instead (here and elsewhere). Otherwise, please add an xml doc comment.

Reply:
Renamed to `trackMembers` and added comment.

---
In reply to: [267962369](https://github.com/dotnet/roslyn/pull/34303#discussion_r267962369) [](ancestors = 267962369)